Dockerfile.test.php7

```diff
@@ -1,7 +1,7 @@
 FROM php:7.2
 
 RUN apt-get update && \
-    apt-get install -y mariadb-client
+    apt-get install -y mariadb-client libzip-dev
```

Comment:
please explain why `libzip-dev` is added

Reply:
Selenium 3 needs a firefox option class to run the test in the browser, in this class also needs zipAchivre class. That is why I add this libzip-dev.